spec/index.html

```diff
@@ -2263,6 +2201,69 @@ <h2>Acknowledgments</h2>
 
 </section>
 
+<section id="section-Changes" class="informative appendix">
+  <h2>Substantive Changes</h2>
+
+<section id="ChangeLog-11" class="informative appendix" >
+  <h2>Substantive changes between RDF 1.0 and RDF 1.1 </h2>
+
+<ul>
+
+ <li>
+    The RDF 1.0 semantics defined simple interpretations relative to a vocabulary.
+  </li>
+
+  <li>
+    In the RDF 1.0 semantics, IL was a total, rather than partial, mapping.
+  </li>
+  
+  <li> The RDF 1.0 specification divided literals into 'plain' literals
+      with no type and optional language tags, and typed literals.
+      Usage has shown that it is important that every literal have a type.
+      RDF 1.1 replaced plain literals without language tags by literals typed with
+      the XML Schema <code>string</code> datatype,
+      and introduced the special type
+      <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string"><code>rdf:langString</code></a>
+      for language-tagged strings.
+      The full semantics for typed literals is given in section [[[#datatypes]]].
+  </li>
+
+  <li>In the RDF 1.0 specification,
+    datatype D-entailment was defined as a <a>semantic extension</a> of RDFS-entailment.
+    In RDF 1.1 it was defined as a direct extension to basic RDF.
+    This is more in conformity with actual usage,
+    where RDF with datatypes is widely used without the RDFS vocabulary.
+    If there is a need to distinguish RDF 1.1 from the RDF 1.0 terminology,
+    the longer phrasing "simple D-entailment" or "simple datatype entailment"
+    should be used rather than "D-entailment".
```

Comment:
```suggestion
    If needed, RDF 1.1's longer phrasing of "simple D-entailment" or "simple 
    datatype entailment" can be used to distinguish from RDF 1.0's shorter
    "D-entailment" terminology.
```

Reply:
I do not view this as a desirable change.

Reply:
I am also a little bit confused by this change: It reads like the term "D-entailment" is only present in RDF 1.0, but that is not the case.

Reply:
The sentence should probably be modified to not be specific to 1.1.  I've put in a suggestion for this.